kafka-client/src/main/java/dev/responsive/kafka/store/ResponsiveWindowStore.java

```diff
@@ -134,21 +134,14 @@ public void init(final ProcessorContext context, final StateStore root) {
   @Override
   public void init(final StateStoreContext context, final StateStore root) {
     try {
+      LOG.info("Initializing state store {} with remote table name {}", name, tableName);
       this.context = asInternalProcessorContext(context);
       this.recordCollector = asRecordCollector(context);
 
       partition = context.taskId().partition();
-      if (partition == 0) {
```

Comment:
was this unintentional? Or are we removing this because Scylla supports LWT?

Reply:
yeah that was intentional - probably should've split it up into two PRs but this was already done for the non-windowed tables. It was only AWS Keyspaces that didn't support it.